src/components/Maps/Maps.js

```diff
@@ -125,8 +191,8 @@ class Maps extends React.Component {
                         intensityExtractor={m => parseFloat(m[2])}
                     />
                     <TileLayer
-                        attribution='&copy; Overzicht'
```

Comment:
I would keep this

Reply:
I think we have to credit the creators of the tile.

Reply:
Ah, in that case I approve the merge request.